package/com.unity.formats.usd/Tests/Editor/PostBuildTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using NUnit.Framework;
+using System.IO;
+using UnityEditor;
+using Assert = UnityEngine.Assertions.Assert;
+
+namespace Unity.Formats.USD.Tests
```

Comment:
A slightly better test would have been to make a runtime test that updates some USD hierarchy(timeline, etc) and assert on some scene state (position at time 10 is 1,2,3, etc).

You get 3 in 1: if build fails, no test,
 if dll's not copied => exception
 if import fails => wrong position

Also test is general purpose: Editor and Standalone.